### DISCUSSION TITLE
Allow viewing rendered markdown files in PR

### DISCUSSION THREAD

**Discussion by zephyap** [community/community#16289] [Pull Requests] [answered]
When using GitHub to store technical documentation, we sometimes create markdown files containing multiple images. This is slightly inconvenient when reviewing the PRs, because we have to click on "View File" for each of the markdown files individually to properly see the changes. A similar problem also exists with tables in the markdown file.

I think it would be nice to have a button in the PR to toggle between viewing the raw diff and viewing a preview of the markdown file (this could even be similar to the preview when editing README files, where you can see the diffs in the preview)

**Comment by laymonage** [comment, marked as answer]
Do you mean this button?

<img width="970" alt="image" src="https://user-images.githubusercontent.com/6379424/169417414-a32c438e-17aa-472a-ab02-1ca988279d2e.png">

**Reply by zephyap** [reply]
Thanks - that was actually what I was after

**Reply by daiplusplus** [reply]
That button really needs a better icon - I never would have thought/expected a button with the "New Document" icon on it would switch diff render modes.

...anyway, I clicked on the button and got this error message:

> _Oh My_
> 
> Sorry, we’re unable to render a diff of this document.

I'm not sure why it decided to do a George Takei impression there...

**Comment by jcollum-nutrien** [comment]
There's a "Preview" button now.

**Reply by mtpultz-dunbar** [reply]
I must be blind I can't find the preview button 😢 this is like when my wife has to rescue me from the fridge and find the ketchup

**Reply by mtpultz-dunbar** [reply]
Oh "View file" hahaha 🙄 thanks @jcollum-nutrien ❤️

**Comment by antazoey** [comment]
what about using the `.yml` based files for issue-templates? I don't see a way to preview those.
